src/utils/utils.ts

```diff
@@ -60,3 +60,23 @@ export const objectValuesByKey = <ValueType>(
 
   return values;
 };
+
+export const splitArrayInChunks = (inputArray, perChunk = 1000) => {
+  const result = inputArray.reduce((resultArray, item, index) => {
+    const chunkIndex = Math.floor(index / perChunk);
+
+    if (!resultArray[chunkIndex]) {
+      resultArray[chunkIndex] = []; // start a new chunk
+    }
+
+    resultArray[chunkIndex].push(item);
+
+    return resultArray;
+  }, []);
+
+  return result;
+};
+
+export const mergeChunkArrays = (inputArray, depth = 1) => {
```

Comment:
This function seems unnecessary. can we just call the the array .flat method directly whereever we are using mergeChunkArrays?

Reply:
Yeah I don't think we need functions that wrap a single function that already exists. Like mergeCHunkArrays only purpose is to call the arrays .flat method but you can just call .flat directly so no need for a wrapper function.